code/modules/mob/living/basic/space_fauna/spider/spiderlings/spiderling.dm

```diff
@@ -62,7 +64,7 @@
 	// it's A-OKAY for grow_as to be null for the purposes of this component since we override that behavior anyhow.
 	AddComponent(\
 		/datum/component/growth_and_differentiation,\
-		growth_time = 10 MINUTES,\
+		growth_time = 1 MINUTES,\
```

Comment:
This may have a significant effect on botany spiderlings, perhaps ones created from weeds need their own subtype which grows more slowly.

Reply:
botany spiderlings taking 10 minutes to grow is a bit ridiculous in all honesty 1 minute seems more appropriate